docs/install/index.md

```diff
@@ -7,13 +7,13 @@ command line tool for creating and updating your instance.
 
 Some system requirements are needed beforehand:
 
-- [Python](https://www.python.org/) (3.6.2 <= supported version < 3.7.0 only)
-- [nodejs](https://nodejs.org) (13.0.0+) (not needed to preview, only needed to develop)
-- [Docker](https://docs.docker.com/) (1.13.0+)
-- [Docker-Compose](https://docs.docker.com/compose/) (1.17.0+)
+- [Python](https://www.python.org/) 3.6.2+ (Python 3.7 has been tested successfully. However, there is still no docker image available and therefore is only available for development).
```

Comment:
and Python 3.8 works too? To be on the safe side I will limit it to `3.6.2 <= supported version < 3.8`